High level diagram of status update process (#758)
italia/eidas-it-wallet-docs: docs/en/credential-revocation.rst

  - The validity status of the Attributes is updated (revocation or suspension).

> In the following diagram is figure out the High-Level status update process for the Authentic Sources.

Reviewer: ```suggestion
The following diagram illustrates the high-level status update process for Authentic Sources.
```

.. only:: format_html

  .. figure:: ./images/svg/status-update-as.svg
    :alt: Status update process for Authentic Sources
    :width: 100%

    Status update process of Authentic Sources

.. only:: format_latex

  .. figure:: ./images/pdf/status-update-as.pdf
    :alt: Status update process process Authentic Sources
    :width: 100%

> The process starts with data or data validity changes that occur in the Authentic Source data. The changes can also be caused by third-party entities different from the Authentic Source, for example in case of illegal activities as notified by Law Enforcement Bodies.

Reviewer: ```suggestion
The process starts with data or data validity changes that occur in the Authentic Source data. Changes can also be initiated by third-party entities other than the Authentic Source, such as when law enforcement agencies report illegal activities.
```

> Once the data changes, the Authentic Source MUST notify the Credential Issuers, to whom it provided such data, using the PDND e-Service :ref:`credential-issuer-endpoint:Notify Update Credential`.

Reviewer: ```suggestion
Once the data changes, the Authentic Source MUST notify the Credential Issuers who received the original data using the PDND e-Service. :ref:`credential-issuer-endpoint:Notify Update Credential`.
```

> Once the data changes, the Authentic Source MUST notify the Credential Issuers, to whom it provided such data, using the e-Service PDND ":ref:`credential-issuer-endpoint:Notify Update Credential`".

Reviewer 2: ```suggestion
Once the data changes, the Authentic Source MUST notify the Credential Issuers, to whom it provided such data, using the PDND e-Service :ref:`credential-issuer-endpoint:Notify Update Credential`.
```

> The Credential Issuer, once it receives the notification, MUST provide for the update of the Credential Status according to the mode defined for the validity mechanism used and MAY send a notification to the User using a registered out-of-band communication channel.

Reviewer: ```suggestion
Upon receiving the notification, the Credential Issuer MUST update the Credential Status according to the validity mechanism's defined mode. The Credential Issuer MAY notify the User through a registered out-of-band communication channel.
```

> The Wallet instance, following periodic checks of the validity status of the stored Digital Credentials, receives the updated status and consequently MUST inform the User of the change of the Credential Status.

Reviewer 2: ```suggestion
The Wallet instance, following periodic checks of the validity status of the stored Digital Credentials, receives the updated status and consequently MUST inform the User in case the Credential Status is changed to INVALID. In case the credential status is modified to UPDATE (resp. 0x03) or ATTRIBUTE_UPDATE (resp. 0x04), the Wallet Instance will proceed to the Re-Issuance of the Digital Credential as described in :ref:`Re-Issuance Flow`.
```

> The process starts with data or data validity changes that occur in the Authentic Source database. The changes can be also caused by third entities different from the Authentic Source, for example in case of illegal activities by the Law Enforcing Bodies.
> 
> Once the data or validity data changes the Authentic Source MUST notify the Credential Issuer to whom it provided such data using the e-Service PDND ":ref:`credential-issuer-endpoint:Notify Update Credential`".
> 
> The Credential Issuer, once it received the notification, MUST provide for the update of the Credential Status according to the mode defined for the validity mechanism used and MAY send a notification to the User using a registered out-of-band communication channel.
> 
> The Wallet instance, following periodic checks of the validity status of the stored Digital Credentials, receives the updated status and consequently MUST inform the User of the change of the Credential Status.

Reviewer: ```suggestion
The process starts with data or data validity changes that occur in the Authentic Source data. The changes can also be caused by third-party entities different from the Authentic Source, for example in case of illegal activities as notified by Law Enforcement Bodies.

Once the data changes, the Authentic Source MUST notify the Credential Issuers, to whom it provided such data, using the e-Service PDND ":ref:`credential-issuer-endpoint:Notify Update Credential`".

The Credential Issuer, once it receives the notification, MUST provide for the update of the Credential Status according to the mode defined for the validity mechanism used and MAY send a notification to the User using a registered out-of-band communication channel.

The Wallet instance, following periodic checks of the validity status of the stored Digital Credentials, receives the updated status and consequently MUST inform the User of the change of the Credential Status.
```

> The Wallet instance, following periodic checks of the validity status of the stored Digital Credentials, receives the updated status and consequently MUST inform the User in case the Credential Status is changed to INVALID. In case the credential status is modified to UPDATE (resp. 0x03) or ATTRIBUTE_UPDATE (resp. 0x04), the Wallet Instance will proceed to the Re-Issuance of the Digital Credential as described in :ref:`Re-Issuance Flow`.

Reviewer: ```suggestion
The Wallet instance, following periodic checks of the validity status of the stored Digital Credentials, receives the updated status. When the Credential Status is changed to INVALID, the Credential Issuer MUST inform the User about this change. In case the Credential status is modified to UPDATE (resp. 0x03) or ATTRIBUTE_UPDATE (resp. 0x04), the Wallet Instance SHOULD proceed to the Re-Issuance of the Digital Credential as described in :ref:`Re-Issuance Flow`.
```
